src/csharp/Microsoft.Spark.Worker/Processor/BroadcastVariableProcessor.cs

```diff
@@ -55,17 +58,34 @@ internal BroadcastVariables Process(Stream stream)
                     else
                     {
                         string path = SerDe.ReadString(stream);
-                        // TODO: Register new broadcast variable.
+                        FileStream fStream = File.Open(path, FileMode.Open, FileAccess.Read);
+                        formatter.Binder = new DeserializationBinder();
+                        var value = formatter.Deserialize(fStream);
+                        fStream.Close();
```

Comment:
Instead of manually closing the `FileStream`, just wrap it in a `using` block.  Then, you don't have to worry about leaks if there's an exception.

Reply:
Makes sense, will do. Thanks!